src/Servers/HttpSys/src/AuthenticationManager.cs

```diff
@@ -32,6 +32,9 @@ internal AuthenticationManager()
         {
         }
 
+        /// <summary>
+        /// When attaching to an existing queue this setting must match the one used to create the queue.
```

Comment:
This should probably be a remark rather than summary.

Reply:
Remarks aren't visible in most situations.